float32 kp_dist
float32 ki_dist
float32 desired_distance
---
bool success
string message